src/lib/transaction.ts

```diff
@@ -85,7 +85,10 @@ const getUserXprvFromServer = async (wallet: GetWalletBackendResponse, userToken
 const createOutputs = (unspentsResponse: ListUnspentsBackendResponse, recipients: Recipient[], changeAddres: string): TxOut[] => {
   const { change, serviceFee } = unspentsResponse
   const changeRecipient: Recipient = { address: changeAddres, amount: btcToSatoshi(new BigNumber(change)) }
-  const serviceRecipient = serviceFee ? [{ address: serviceFee.address, amount: btcToSatoshi(new BigNumber(serviceFee.amount)) }] : []
+  const serviceRecipient = serviceFee ? [{
+    address: serviceFee.address,
+    amount: btcToSatoshi(new BigNumber(serviceFee.amount))
```

Comment:
dalej `btcToSatoshi`?

Reply:
to jest co innego - tu się tylko formatowanie zmieniło